### PR TITLE
Add github ids for comments, tasks and projects

### LINKED ISSUE #798: Add Github ID fields for all related models
# Problem

Need to add a `github_id` field for any model that will be connected to a related github model

## Subtasks
- [x] Add user field (covered in #792)
- [x] Add task field
- [x] Add project field
- [x] Add comment field

Progress on: # Milestone 11

### PR DESCRIPTION
**Pull request by npendery**
# What's in this PR?

Adds github_ids for multiple models

## References
Fixes #798

### CODE REVIEW

**Comment by securingsincity** [comment]
Do we want to include the relevant changeset in this PR?

**Comment by npendery** [comment]
@securingsincity yeah, wouldn't hurt. have it in my webhooks PR but dont see why not just adding it in this one 👍

File: web/models/comment.ex [hunk 1]
```diff
@@ -33,4 +34,13 @@ defmodule CodeCorps.Comment do
     |> assoc_constraint(:task)
     |> assoc_constraint(:user)
   end
+
+  @doc """
+  Builds a changeset for creating a comment that has a connected GitHub comment.
+  """
+  def github_create_changeset(struct, params) do
```
Comment (securingsincity) [inline web/models/comment.ex:41]: can you add tests for these changsets as well?

File: web/models/project.ex [hunk 2]
```diff
@@ -73,6 +74,15 @@ defmodule CodeCorps.Project do
   end
 
   @doc """
+  Builds a changeset for creating a project that has a connected GitHub repo.
+  """
+  def github_create_changeset(struct, params) do
```
Comment (securingsincity) [inline web/models/project.ex:79]: can you add tests for these changsets as well?

File: web/models/task.ex [hunk 3]
```diff
@@ -51,6 +52,15 @@ defmodule CodeCorps.Task do
     |> put_change(:status, "open")
   end
 
+  @doc """
+  Builds a changeset for creating a task that has a connected GitHub issue.
+  """
+  def github_create_changeset(struct, params) do
```
Comment (securingsincity) [inline web/models/task.ex:58]: can you add tests for these changsets as well?

**Review by securingsincity — CHANGES_REQUESTED** [review]

**Review by begedin — CHANGES_REQUESTED** [review]
Other than the missing changeset tests, this looks good.

There's a possibility we might need to extract these ids into actual models containing additional github information at some point, but for now, I think this is enough.

**Comment by npendery** [comment]
All set @begedin

**Review by begedin — APPROVED** [review]

**Pull request merged**